templates/zerver/help/restrict-permissions-of-new-members.md

```diff
@@ -12,24 +12,20 @@ past a certain **waiting period** threshold. After that they are **full members*
 You can configure how long the waiting period is, as well as which actions require
```

Comment:
Let's change "large Zulips" -> "large Zulip organizations"

Reply:
Updated.

There's one other help center page that uses "Zulips" - `configure-authentication-methods`. Do we want to mark that as a follow-up change as well?

Reply:
Yeah, that follow-up would be great!